src/maintenance/update_last_usage.py

```diff
@@ -0,0 +1,58 @@
+import os
+from datetime import datetime as dtime
+
+import delphi.operations.secrets as secrets
+import mysql.connector
+import redis
+
+REDIS_HOST = os.environ.get("REDIS_HOST", "delphi_redis")
+REDIS_PASSWORD = os.environ.get("REDIS_PASSWORD", "1234")
+LAST_USED_KEY_PATTERN = "*LAST_USED*"
+
+
+def redis_get_last_time_used(redis_cli, pattern: str):
+    keys = redis_cli.keys(pattern=pattern)
```

Comment:
i think its safe to use this directly instead of passing it in as an argument
```suggestion
def redis_get_last_time_used(redis_cli):
    keys = redis_cli.keys(pattern=LAST_USED_KEY_PATTERN)
```